src/Components/App/App.tsx

```diff
@@ -1,22 +1,81 @@
-import './App.css';
-import NavBar from '../NavBar/NavBar';
-import { Route, Routes } from 'react-router-dom';
-import Homepage from '../Homepage/Homepage';
+import "./App.css";
+import React from "react";
+import { useEffect, useState } from "react";
```

Comment:
I believe you can get rid of line 2 altogether since you're using React 18.  Or at the very least you can combine lines 2 and 3.

Reply:
Typescript was yelling at us when it was removed so not sure if that’s just a quirk of TS 😭